docs/getting-started.rst

```diff
@@ -1,39 +1,74 @@
+We can configure the spider in the toolbox the right. You may notice the URL we entered has been added as a start page. These start pages act as seeds for the crawl and the spider will visit these first. If the site we're using requires JavaScript, we can enable it under the Crawling section.
+
+Now that we've created our spider, we need to define the data we want to extract. We do this using `samples <samples>`_.
 
-Annotating a page
+Creating a sample
 -----------------
 
+Portia acts like a web browser, so we can navigate between pages as we would normally. Let's navigate to a page we want to extract data from and click ``Annotate this page`` to create a sample.
+
 .. image:: _static/portia-annotation.png
     :alt: Annotating the page
 
-You will now be able to define annotations by highlighting or clicking elements on the page. When annotating, a context menu will appear allowing you to map an element's attribute or content to a particular item field. Should you want to add a new item field without having to go into the item editor, you can use the ``-create new-`` option in the field drop down to create a new field. If you want to mark an element as having to exist on the page without storing its data, you can select ``-just required-`` instead of a field. It's important to note when using ``-just required-``, only the existence of the element will be checked rather than its content.
+A sample is a template of a page made up of annotations. Annotations define what data should be extracted and where to.
+
+We'll come back to annotations in a moment, but first we need to create our item.
+
+Items
+-----
+
+An :ref:`item <items>` in Portia is a record of data e.g. a product, a location that's extracted during the crawl. We want to define the schema for the item we're going to be extracting. A field is simply an item's attribute, so for example a book would have a `title` field, an `author` field etc.
+
+We should ideally define all of the fields for data we want to extract before annotating. While we can create and modify an item's fields through the annotation menu, it's easier to use the item editor.
 
 .. image:: _static/portia-item-editor.png
     :alt: Items editor
 
-Once you are finished annotating, you can then mark any fields that are required by going into the item editor under ``Extracted item``. As mentioned earlier, if the item appears in several locations and some fields differ despite being the same item, you can also tick ``Vary`` on any relevant fields to exclude them from being used to detect duplicate items.
+We need to think about the data we want to extract, and add the relevant fields. We can mark fields as required and any items extracted that are missing them will be discarded. Marking fields as vary means they won't be taken into account when checking for duplicates, more on that `here <>`. 
```

Comment:
Fields can also be added while annotating, this makes it sound like you need to predefine your items when realistically you can just create a new item and then add all of the fields while annotating